classifiers/fact_relevance/gpt_cross_encoder/code_snippet_refinery.md

```diff
@@ -0,0 +1,46 @@
+```python
+import openai
+import os 
+
+ATTRIBUTE: str = "text" 
+FACT: str = "fact"
+API_KEY: str = "<API_KEY_GOES_HERE>"
+API_BASE: str = "https://api.openai.com/v1"
+API_TYPE: str = "open_ai" # or 'azure'
+API_VERSION: str = None
+ENGINE: str = None
```

Comment:
What do you think of adding similar info here as in the common snippet? With that many parameters it could be hard for the user to figure out, when which needs to be set

Reply:
Yes, I have added some comments. Do you think this is sufficient then?

Reply:
looks good